.gitignore

```diff
@@ -19,11 +19,12 @@
 .env.production.local
 .space
 .env
+.netlify
```

Comment:
lol remove this and `.netlify` you might have added from install the netlify-cli